### PR TITLE
fix(ci): missing long_description in build

### PR DESCRIPTION
**Pull request by vicwomg**
missing long_description in build

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -3,6 +3,15 @@ name = "pikaraoke"
 version = "1.4.0"
 description = "Youtube-based Karaoke machine for Raspberry Pi, OSX, Windows, and Linux"
 authors = ["Vic Wong"]
+readme = "docs/README.md"
```
Comment (mariugul) [inline pyproject.toml:6]: I think missing this is what caused the error.

**Review by mariugul — APPROVED** [review]
I think we should try with this.

**Comment by vicwomg** [comment]
I have no idea what I'm doing with commit messages. These conventions are new to me. I'll read up but a quick rundown of the most common use cases would be much appreciated

**Comment by mariugul** [comment]
> I have no idea what I'm doing with commit messages. These conventions are new to me. I'll read up but a quick rundown of the most common use cases would be much appreciated

Here's a quick intro. The most used are `feat` `fix` `refactor` and a short summary.
Sometimes you want a body too.

### How to Write Commit Messages with Conventional Commits

The **Conventional Commits** format has a simple structure to make commit messages clear and consistent.

#### Structure

type(scope): subject

- **Type**: Describes the category of the change (e.g., `feat`, `fix`).
- **Optional Scope**: Indicates the part of the codebase affected (e.g., `scope`).
- **Subject**: Brief summary of the change.

#### Common Types

- **feat**: A new feature
- **fix**: A bug fix
- **docs**: Documentation changes
- **style**: Code style changes (white-space, formatting, etc.)
- **refactor**: Code refactoring without changing any functionality
- **test**: Adding or updating tests
- **chore**: Maintenance tasks (e.g., build process, package manager configs)

#### Examples

- `feat(login): add password reset feature`
- `fix(auth): correct validation for email field`
- `docs(readme): update setup instructions`
- `refactor(utils): simplify date formatting function`
- `style(header): format code according to ESLint rules`
- `test(api): add tests for user endpoints`
- `chore(ci): update CircleCI configuration`

##### Full Body Example

```text
feat: add two-factor authentication

Added two-factor authentication (2FA) to enhance account security. Users can now enable 2FA from their profile settings.

- Implemented 2FA using time-based one-time password (TOTP) algorithm.
- Updated the user model to store 2FA settings.
- Added a new API endpoint for 2FA setup and validation.
- Modified the login process to require a 2FA code if enabled.

BREAKING CHANGE: Users are now required to set up 2FA upon their next login to ensure compliance with new security policies.

Closes #123, #456
```

#### Tips

- Keep the **subject** concise (50 characters or less) and use the **imperative mood** (e.g., "add", "fix", "update").
- Optionally, add a more detailed body and footer if needed for further explanation or to reference issues.

**Pull request merged**